socrata-metadata/socrata_metadata_pub.py

```diff
@@ -0,0 +1,208 @@
+"""
+Extract metadata about public datasets owned and maintained by DTS
+and publishes the data in another dataset.
+"""
+from datetime import datetime
+import pytz
```

Comment:
it looks like `pytz` is missing from the `requirements.txt`. maybe the script works because your other dependencies use the same package 🤷‍♂️